From 57abcd52e9210d04e971a3d82124b406dfe8781e Mon Sep 17 00:00:00 2001
From: goumin <min.gou@spacemit.com>
Date: Tue, 19 Nov 2024 16:25:42 +0800
Subject: [PATCH 1291/1451] The debug parameter in bootargs overrides loglevel,
 making it ineffective. debug should be added to U-Boot's environment bootargs
 or replaced with loglevel=8.

Change-Id: Ie0376f9e724e701685b571c1104017c02e55c7be
---
 arch/riscv/boot/dts/spacemit/k1-x_FusionOne.dts          | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts            | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts          | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Card.dts          | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts            | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper-mini-4g.dts | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper.dts         | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_MUSE-Pi.dts            | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_ZT001H.dts             | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_deb1.dts               | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_deb2.dts               | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_evb.dts                | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_hs450.dts              | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_kx312.dts              | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_lpi3a.dts              | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_milkv-jupiter.dts      | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_mingo.dts              | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_orangepi-rv2.dts       | 2 +-
 arch/riscv/boot/dts/spacemit/k1-x_uav.dts                | 2 +-
 arch/riscv/boot/dts/spacemit/m1-x_milkv-jupiter.dts      | 2 +-
 20 files changed, 20 insertions(+), 20 deletions(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x_FusionOne.dts b/arch/riscv/boot/dts/spacemit/k1-x_FusionOne.dts
index 0874359a491f..be690425cf88 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_FusionOne.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_FusionOne.dts
@@ -117,7 +117,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts b/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
index 1be8647964e2..d6dcf25092d2 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MINI-PC.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
index 85e70e676e9a..5690722c9bed 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Book.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Card.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Card.dts
index 2942122e7968..a438ed1b2d21 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Card.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Card.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
index 7709daad504c..094fa3646abe 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-N1.dts
@@ -112,7 +112,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper-mini-4g.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper-mini-4g.dts
index a695fcc66bf7..89e1953dca82 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper-mini-4g.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper-mini-4g.dts
@@ -119,7 +119,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper.dts
index 572f97ab4cbb..31b01e0fceba 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Paper.dts
@@ -119,7 +119,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Pi.dts b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Pi.dts
index 13393ae8d670..9107d43c3091 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Pi.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_MUSE-Pi.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_ZT001H.dts b/arch/riscv/boot/dts/spacemit/k1-x_ZT001H.dts
index 070b8996e898..246f6d712b4a 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_ZT001H.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_ZT001H.dts
@@ -118,7 +118,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts b/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
index 2cab7ea9f545..30c03d0f803e 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_deb1.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts b/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
index e9cd8e7d4690..054ebac515c1 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_deb2.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_evb.dts b/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
index faa2fb213b1f..66f2a3417a32 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_evb.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts b/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
index 37a18ed7dc99..8e10e49f7a90 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_hs450.dts
@@ -114,7 +114,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts b/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
index 84ad7eaf41dc..f80ceeaf9ff1 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_kx312.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_lpi3a.dts b/arch/riscv/boot/dts/spacemit/k1-x_lpi3a.dts
index 2a8c886881bc..03583737c66b 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_lpi3a.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_lpi3a.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_milkv-jupiter.dts b/arch/riscv/boot/dts/spacemit/k1-x_milkv-jupiter.dts
index f8f4720a7cd2..09a6f2699a41 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_milkv-jupiter.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_milkv-jupiter.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts b/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
index f59adb3e379c..53a4408113a6 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_mingo.dts
@@ -111,7 +111,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8,initcall_debug=1 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8,initcall_debug=1 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_orangepi-rv2.dts b/arch/riscv/boot/dts/spacemit/k1-x_orangepi-rv2.dts
index d6ad16529085..1ddfc40fe764 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_orangepi-rv2.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_orangepi-rv2.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/k1-x_uav.dts b/arch/riscv/boot/dts/spacemit/k1-x_uav.dts
index 5189dd9000e3..2535fa9f6bea 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x_uav.dts
+++ b/arch/riscv/boot/dts/spacemit/k1-x_uav.dts
@@ -115,7 +115,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
diff --git a/arch/riscv/boot/dts/spacemit/m1-x_milkv-jupiter.dts b/arch/riscv/boot/dts/spacemit/m1-x_milkv-jupiter.dts
index e6d74c2d1672..cfabd0687446 100644
--- a/arch/riscv/boot/dts/spacemit/m1-x_milkv-jupiter.dts
+++ b/arch/riscv/boot/dts/spacemit/m1-x_milkv-jupiter.dts
@@ -116,7 +116,7 @@ dpu_resv: dpu_reserved@2ff40000 {
 	};
 
 	chosen {
-		bootargs = "earlycon=sbi console=ttyS0,115200n8 debug loglevel=8 swiotlb=65536 rdinit=/init";
+		bootargs = "earlycon=sbi console=ttyS0,115200n8 loglevel=8 swiotlb=65536 rdinit=/init";
 		stdout-path = "serial0:115200n8";
 	};
 
-- 
2.47.0

